cerbero/packages/wix_packager.py

```diff
@@ -297,13 +356,16 @@ def __init__(self, wix_prefix, with_wine, extra=''):
             self.options['wine'] = ''
             self.options['q'] = ''
 
-    def compile(self, objects, msi_file, output_dir, merge_module=False):
+    def compile(self, objects, msi_file, output_dir, merge_module=False, ext=None):
         self.options['objects'] = ' '.join(objects)
         self.options['msi'] = msi_file
-        if merge_module:
-            self.options['ext'] = 'msm'
+        if ext:
+            self.options['ext'] = ext
```

Comment:
what is ext? if it's extension Can you add that rename the variable with that to be more clear.